tests/server/test_load.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from NIPTool.exeptions import NIPToolError, FileValidationError
+
+
+def test_user(mock_app):
+    # GIVEN the following request data:
+    data = dict(email='maya.papaya@something.se', name="Maya Papaya", role="RW")
+
+    # WHEN running the user request with the data
+    mock_app.test_client().post('/user', data=data)
```

Comment:
Looks like the POST request can be sent from anywhere? So perhaps it should not redirect to the request.referrer but it should instead return a json object with a message and a success status (200) ?

Reply:
Here you mean the test should check the status_code?

Reply:
If you check the status code (and the message) it would increase the coverage

Reply:
Okej. Thank you!

Reply:
assert response.status_code == 200
 resp_data = json.loads(response.data)
 assert resp_data["message"] == "Whatever your message should contain"